### PR TITLE
Raceid Fix

### PR DESCRIPTION
**Pull request by mtekman**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Updates to plots

### CODE REVIEW

File: tools/raceid/raceid_filtnormconf.xml [hunk 1]
```diff
@@ -65,22 +65,20 @@ Rscript '$__tool_directory__/scripts/$bin' '$userconf' 2> '$outlog' > /dev/null
...
-        </section>                    
+        </section>
     </inputs>
     <outputs>
         <data name="outpdf" format="pdf" label="${tool.name} on ${on_string}: PDF Report" />
         <data name="outrdat" format="rdata" label="${tool.name} on ${on_string}: RDS" />
-        <data name="outlog" format="txt" label="${tool.name} on ${on_string}: Log" >
-            <filter>use_log</filter>
-        </data>
+         <data name="outlog" format="txt" label="${tool.name} on ${on_string}: Metrics"  />
```
Comment (bgruening) [inline tools/raceid/raceid_filtnormconf.xml]: indentation

Comment (bgruening) [reply tools/raceid/raceid_filtnormconf.xml]: why not keeping the optional parameter?

File: tools/raceid/raceid_filtnormconf.xml [hunk 2]
```diff
@@ -108,6 +106,14 @@ Rscript '$__tool_directory__/scripts/$bin' '$userconf' 2> '$outlog' > /dev/null
             </section>
             <output name="outpdf" value="intestinal.filter.pdf" compare="sim_size" delta="50" />
         </test>
+        <test>
```
Comment (bgruening) [inline tools/raceid/raceid_filtnormconf.xml]: You can use expected_num_outputs to test for the number of expected outputs.
https://docs.galaxyproject.org/en/latest/dev/schema.html#id77

File: tools/raceid/raceid_filtnormconf.xml (hunk 1 above)
Comment (mtekman) [reply tools/raceid/raceid_filtnormconf.xml]: It has useful metrics, users are quite likely to want to know what is being shown in the log file.

File: tools/raceid/raceid_filtnormconf.xml [hunk 3]
```diff
@@ -79,6 +79,7 @@ Rscript '$__tool_directory__/scripts/$bin' '$userconf' 2> '$outlog' > /dev/null
             <param name="intable" value="use.intestinal" />
             <output name="outpdf" value="intestinal.filter.pdf" compare="sim_size" delta="50" />
             <output name="outlog" value="intestinal.filter.log" />
+            <expect_num_outputs value="3" />
         </test>
         <test>
```
Comment (bgruening) [inline tools/raceid/raceid_filtnormconf.xml]: <test expect_num_outputs="3"> seems to be easier. Don't forget the indentation problem.

**Comment by mtekman** [comment]
@bgruening I am getting weird errors in the travis build where `expect_num_outputs` is not expected.
All tests are green locally however....

https://travis-ci.org/galaxyproject/tools-iuc/jobs/520293700

**Pull request merged**